nodes/src/nodes/preprocessor_langchain/langchain.py

```
@@ -74,6 +75,8 @@ def _parseSeparators(self, user_input: str):
         """
         Parse a quoted, comma-separated string into a list of actual separator strings.
         """
+        if len(user_input) > self._MAX_SEPARATOR_INPUT_LEN:
+            raise ValueError(f'Separator input too long ({len(user_input)} chars, max {self._MAX_SEPARATOR_INPUT_LEN})')
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)